workflow/usage.rst

```diff
@@ -138,24 +138,33 @@ like this:
 
     class BlogPost
     {
-        // This property is used by the marking store
         public $currentPlace;
         public $title;
         public $content;
     }
 
+.. tip::
+
+    The ``type`` (default value ``single_state``) and ``arguments`` (default value ``marking``)
```

Comment:
For 4.3+ this need to be removed no?
As single/multiple state are deprecated/removed in favor of method